src/swift/DispatchStubs.cc

```diff
@@ -11,11 +11,11 @@
 //===----------------------------------------------------------------------===//
 
 #include <dispatch/dispatch.h>
-#include <objc/runtime.h>
 #include <stdio.h>
 
 #define DISPATCH_RUNTIME_STDLIB_INTERFACE __attribute__((__visibility__("default")))
 
+#if 0
```

Comment:
I'd like to rename the file anyways.  Having both a Dispatch.swift and a Dispatch.mm complicates the build (clash of Dispatch.o).  It could be DispatchStubs.mm I suppose, but since DispatchStubs.mm != Dispatch.mm, switching to .cc (or .c) seemed like it was worth doing.  Don't have strong feelings on .mm/.m/.cc/.c however.